script/cideploy

```diff
@@ -2,48 +2,42 @@
 set -x
 set -e
 
-: ${BRANCH:=master}
-: ${TARGET_REPO=CHTC/chtc.github.io.git}
+: "${BRANCH:=master}"
+: "${TARGET_REPO=CHTC/chtc.github.io.git}"
 JEKYLL_OUTPUT_FOLDER=_site
 
-echo "$TRAVIS_PULL_REQUEST"
+echo "$GITHUB_EVENT_NAME"
 
 commitmessage=$(mktemp)
 trap 'rm -f "$commitmessage"' EXIT
 
-if [[ $TRAVIS_PULL_REQUEST == false || $TRAVIS != true ]]; then
-    git log -1 --pretty=format:"Travis $TRAVIS_BUILD_NUMBER: \"%s\" by %an [%h] pushed to GitHub Pages
+git log -1 --pretty=format:"GHA $GITHUB_RUN_NUMBER: \"%s\" by %an [%h] pushed to GitHub Pages
 
 %b" > "$commitmessage"
 
-    if [[ -z $NO_PUSH ]]; then
-        mkdir -p ~/.ssh
-        echo "github.com,207.97.227.239 ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==" >> ~/.ssh/known_hosts
-        openssl aes-256-cbc -K $encrypted_641cbba37704_key -iv $encrypted_641cbba37704_iv -in deploy_key.enc -out deploy_key -d
-        mv deploy_key ~/.ssh/id_rsa
-        chmod 0600 ~/.ssh/id_rsa
-    fi
-
-    echo -e "Starting to deploy to Github Pages\n"
-
-    if [[ $TRAVIS == true ]]; then
-        git config --global user.email "travis@travis-ci.org"
-        git config --global user.name "Travis"
-    fi
-
-    #using token clone target repository
-    git clone --quiet --branch=$BRANCH git@github.com:$TARGET_REPO built_website > /dev/null
-    #go into directory and copy data we're interested in to that directory
-    mkdir -p built_website
-    cd built_website
-    rsync -av --delete --exclude-from=../script/rsync_exclude_list "../$JEKYLL_OUTPUT_FOLDER/" .
-    # Rename .html -> .shtml for generated files
-    rename --force --verbose 's/\.html$/.shtml/' ./*.html
-    #add, commit and push files
-    git add -f -A
-    git commit -F "$commitmessage"
-    if [[ -z $NO_PUSH ]]; then
-        git push -fq origin $BRANCH > /dev/null
-    fi
-    echo -e "Deploy completed\n"
+echo -e "Starting to deploy to Github Pages\n"
+
+if [[ $GITHUB_ACTIONS == true ]]; then
+    git config --global user.email "actions@github.com"
+    git config --global user.name "GitHub Actions"
```

Comment:
Why set the email to one we don't control? I can understand setting the name like that but I'm not sure I get why we'd want to set the email this way.

Reply:
We need to set the email to _something_ or git won't let us commit. For the travis build we set it to "travis@travis-ci.org". If you have another idea, I can use that.

Reply:
Mmk, that's fine by me then. Though I do wonder how `mkdocs` does it https://github.com/opensciencegrid/docs/commit/827b4d851ab659bc231a16fc0569900a25298a9b.